src/stories/Header.stories.ts

```
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+
+import { Header } from "./Header";
```

ESLint

[error] Missing file extension for "./Header"

(import/extensions)